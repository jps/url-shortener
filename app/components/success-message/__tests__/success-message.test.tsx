import { render } from "@testing-library/react";
import { SuccessMessage } from "../";

describe("SuccessMessage should", () => {
  it("render as expected", () => {
    const { asFragment } = render(
      <SuccessMessage shortenedUrl="https://example.com" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
