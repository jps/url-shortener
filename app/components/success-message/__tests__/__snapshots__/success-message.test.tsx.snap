// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessMessage should render as expected 1`] = `
<DocumentFragment>
  <div
    class="success-message"
  >
    <h2
      class="success-message__title"
    >
      Your URL has been shortened
    </h2>
    <p
      class="success-message__text"
    >
      <a
        href="https://example.com"
      >
        https://example.com
      </a>
    </p>
  </div>
</DocumentFragment>
`;
